src/main/java/com/puppycrawl/tools/checkstyle/gui/ListToTreeSelectionModelWrapper.java

```diff
@@ -39,10 +39,10 @@
     /** Set to true when we are updating the ListSelectionModel. */
     protected boolean updatingListSelectionModel;
     /** JTreeTable to perform updates on */
-    private final JTreeTable jTreeTable;
```

Comment:
can you provide some reliable sources explaining why names like `jTreeTable` are invalid?